test/jdk/com/sun/jdi/JdwpAttachTest.java

```diff
@@ -82,10 +85,12 @@ public static void main(String[] args) throws Exception {
         }
 
         // by using "localhost" or empty hostname
-        // we should be able to attach to both IPv4 and IPv6 addresses (127.0.0.1 & ::1)
+        // we should be able to attach to both IPv4 and IPv6 addresses (127.0.0.1 & ::1).
```

Comment:
As long as you're touching this comment, start it with "By" and put more of the comment on the first line.

Also, the comment is a bit misleading now. You can attach to both, but you only try the one that is preferred.